### ISSUE TITLE
Offline QA doesn't respond.

### ISSUE THREAD

**Issue by OriginalDecode** [logancyang/obsidian-copilot#129]
Entirely unsure if I've set something up incorrectly so I apologize in advance.   
Love the work you've done and I'm very excited to use this.

I've got a docker environment running in WSL using the bert-embedding with cors enabled, also enabled debug so that I can see what it's attempting to do. 
I can see that the question I ask about my note gets recieved by the LocalAI server, but there's no response. 
The correct model is loaded and it reads my note as I expect, at least that's what it looks like. 
But it just doesn't respond to me

The message I see is as follow

> -- READING IN THE NOTE ABOVE HERE --
2023-08-17 00:23:28 Describe the braindump for me
2023-08-17 00:23:28 10:23PM DBG Loading model bert-embeddings from bert-MiniLM-L6-v2q4_0.bin
2023-08-17 00:23:28 10:23PM DBG Model already loaded in memory: bert-MiniLM-L6-v2q4_0.bin
2023-08-17 00:23:28 10:23PM DBG Sending chunk: {"object":"chat.completion.chunk","model":"text-embedding-ada-002","choices":[{"index":0,"delta":{"role":"assistant","content":""}}],"usage":{"prompt_tokens":0,"completion_tokens":0,"total_tokens":0}}

The last line makes me think that it doesn't actually do anything when it arrives at the server, so I am confused if I'm missing anything in the configuration or missed a step in the step-by-step guide.

Using LocalAI - Set to QA Mode - pointing to http://localhost:8080/v1 - Docker env running - preloaded models - request received but no response from the server.

I've also tried to run with the rebuild step, I've restarted the plugin.
The plugin debug info just says

> *** DEBUG INFO ***
user message: Describe the braindump for me
model: text-embedding-ada-002
temperature: 0.7
maxTokens: 1000
system message: You are Obsidian Copilot, a helpful assistant that integrates AI to Obsidian note-taking.
chat context turns: 3
embedding provider: localai
Refused to set unsafe header "User-Agent"

**Comment by logancyang** [comment]
@OriginalDecode Thanks for reporting, this feature is experimental so definitely don't hesitate to report any issue you see!

Let me try to reproduce this on my WSL. The local QA is quite slow at this point, I need to dig in more.

**Comment by Sokole1** [comment]
Hi @OriginalDecode, it looks like you are trying to use text-embedding-ada-002 as your chat model which is an embedding provider and not a chat model, can you try downloading and changing your model to `model: luna-ai-llama2-uncensored.ggmlv3.q4_K_S.bin` as in the example here? https://github.com/logancyang/obsidian-copilot/blob/master/localai_setup.md#how-to-install-models-and-test-the-api. I was able to replicate the no response error when I changed my model to `bert-MiniLM-L6-v2q4_0.bin`, the embedding model.

**Comment by logancyang** [comment]
@OriginalDecode As @Sokole1 pointed out, looks like you typed in `text-embedding-ada-002` as your local chat model.

I'm thinking about adding the ability to pick from downloaded models to replace the text box for model name. But still, it's going to require the user to make the correct selection there.

**Comment by OriginalDecode** [comment]
@Sokole1 @logancyang thank you for pointing out the chat model was wrong, I fixed that, and turned the plugin off and back on again and tried once more , sadly it did not resolve the issue. 

However removing PRELOAD_MODELS from the LocalAI .env file did. I added it in an attempt to make the responses a bit faster by skipping loading the models into memory.

I assume that the problem was there wasn't enough RAM available and that's why I didn't get a reply.

In conclusion.

The LocalAI model setting in Copilot should **not** be changed to the embeddings provider model name (this was my assumption that I was supposed to do this). 
Maybe there's room to add a 
**NOTE** don't set the embedding provider as the LocalAI Model
so people don't repeat this mistake?

And as always you gotta make sure you have enough RAM available. I assumed I had enough.

Thank you for your assistance! And keep up the good work.
